lib/matplotlib/tests/test_backend_tk.py

```diff
@@ -37,6 +37,10 @@ def _isolated_tk_test(success_count, func=None):
         sys.platform == "linux" and not _c_internal_utils.display_is_valid(),
         reason="$DISPLAY and $WAYLAND_DISPLAY are unset"
     )
+    @pytest.xfail(  # GitHub issue #23094
+        reason="Tk version mismatch on OSX CI",
+        condition=sys.platform == 'darwin'
+    )
```

Comment:
```suggestion
    @pytest.xfail(  # GitHub issue #23094
        sys.platform == 'darwin',
        reason="Tk version mismatch on OSX CI"
    )
```

Reply:
Commiting this to confirm that it fixes it. Feel free to force push without lease to get rid of it if preferred.